apps/astro/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "astro-particles-demo",
   "type": "module",
-  "version": "2.10.0",
+  "version": "3.40.0",
```

Comment:
Version should be 3.0.0, this package has it's own version, separate from the engine.